devmgmtV2/controllers/ssid.controller.js

```diff
@@ -0,0 +1,43 @@
+"use strict"
+
+const path = require('path');
+const exec = require('child_process').exec;
+
+
+const setSSID = (req, res) => {
+
+    const ssid = req.body['ssid'].trim();
+
+    let responseData = {
+        ssidSetSuccessful : true,
+        msg : 'Successfully set the ssid id to ' + ssid
+    }
+
+    if (typeof ssid === "undefined" || ssid.length < 1) {
+        responseData.ssidSetSuccessful = false;
+        responseData.msg = "Empty ssid not allowed!";
+
+        res.status(200).json(responseData);
```

Comment:
add return statement or else statement, otherwise the code will continue downwards